textbook-market/lib/mongodb.js

```diff
@@ -1,34 +1,33 @@
 /*
     Example taken from Vercel's github, https://github.com/vercel/next.js/blob/canary/examples/with-mongodb/lib/mongodb.ts
 */
+import 'server-only'
```

Comment:
can we drop the changes to this file altogether?

Reply:
Sure